schema/restDataSource.schema.json

```diff
@@ -33,6 +33,57 @@
               "type": "string",
               "enum": ["GET", "POST"]
             },
+            "pagingScheme": {
+              "type": "object",
+              "properties": {
+                "type": {
+                  "type": "string",
+                  "enum": ["offsetAndLimit", "pageBased"]
+                },
+                "request": {
+                  "type": "object",
+                  "properties": {
```

Comment:
I wonder if we should define separate objects for each of the distinct types here and then reference those in some sort of union (assuming that's possible in JSONShema).  Doesn't have to happen now but let's think about this as the other paging schemes come togehter.

Reply:
Yes thanks, `oneOf` allows us to define subschemas.